src/Dash/Router/Parser/ParserManager.php

```diff
@@ -20,8 +20,8 @@ class ParserManager extends AbstractPluginManager
     protected $shareByDefault = false;
 
     protected $factories = [
-        'pathsegment'     => 'Dash\Router\Http\Parser\PathSegmentFactory',
-        'hostnamesegment' => 'Dash\Router\Http\Parser\HostnameSegmentFactory',
+        'pathsegment'     => Dash\Router\Parser\PathSegmentFactory::class,
+        'hostnamesegment' => Dash\Router\Parser\HostnameSegmentFactory::class,
```

Comment:
Well the rpoblem with FQCN is that you can hit the number of characters limit, with short names we will be sure we won't have this problem :).